packages/frontend/src/components/Bid/WinBid/WinBidFlow.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react'
+import { AuctionTransaction } from 'src/components/Auction/AuctionTransaction'
+import { BidFlowSteps } from 'src/components/Bid/BidFlowEnum'
+import { WinBidForm } from 'src/components/Bid/WinBid/WinBidForm'
+import { WinOptions } from 'src/components/Bid/WinBid/WinFlowEnum'
+import { Transactions } from 'src/components/Transaction/TransactionEnum'
+import { Bid } from 'src/models/Bid'
+
+interface WinBidFlowProps {
+  userBid: Bid
+}
+
+export const WinBidFlow = ({ userBid }: WinBidFlowProps) => {
+  const [view, setView] = useState<BidFlowSteps>(BidFlowSteps.Placing)
+  const WinBidAction = Transactions.Withdraw
+
+  return (
+    <>
+      {view === BidFlowSteps.Placing ? (
+        <WinBidForm bid={userBid.amount} setView={setView} win={WinOptions.Ticket} />
+      ) : (
+        <AuctionTransaction action={WinBidAction} amount={userBid.amount} view={view} setView={setView} />
```

Comment:
There should be different text in transaction stepper when we finalize withdrawn. 
Right now it's: 
`Initiate and confirm bid transaction in your wallet.` and should be something like:
`Initiate and confirm withdrawn transaction in your wallet.`

I think that [transactionSteps](https://github.com/TrueFiEng/devcon-raffle/blob/33268f600591cb43efafc027a02f18a00a0b4df1/packages/frontend/src/components/Transaction/TransactionStepper.tsx#L14) should change description based on passed action.